Content.Shared/PAI/PAIComponent.cs

```diff
@@ -31,6 +31,13 @@ public sealed partial class PAIComponent : Component
     [DataField("midiAction", serverOnly: true)] // server only, as it uses a server-BUI event !type
     public EntityUid? MidiAction;
 
+    [DataField("mapActionId", serverOnly: true,
+        customTypeSerializer: typeof(PrototypeIdSerializer<EntityPrototype>))]
+    public string? MapActionId = "ActionPAIOpenMap";
```

Comment:
Use the new ProtoId please.

Reply:
I'll try my best based on https://docs.spacestation14.com/en/general-development/codebase-info/conventions.html?highlight=protoID#prototype-data-fields

If it's wrong after the update, I'll need some pointers on where I'm going wrong. This is my first contrib and I'm brand new to C# but have professionally used other languages a lot (Python)